packages/bitcore-wallet-service/src/lib/chain/eth/index.ts

```diff
@@ -467,9 +467,16 @@ export class EthChain implements IChain {
       address = Web3.utils.toChecksumAddress(tx.abiType.params[0].value);
       amount = tx.abiType.params[1].value;
     } else if (tx.abiType && tx.abiType.type === 'MULTISIG' && tx.abiType.name === 'confirmTransaction') {
+      let amount = 0,
```

Comment:
amount and address are being redefined here, and vscode is saying these definition vars are unused...